.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description: Create a bug report for rust-libp2p
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for filing a bug report!
+
+        For security related issues please file a private security vulnerability report at https://github.com/libp2p/rust-libp2p/security/advisories/new or reach out to security@libp2p.io. Please do not file a public issue on GitHub.
```

Comment:
There already is a link for security vulnerabilities in the issue selection, see: 

![image](https://github.com/libp2p/rust-libp2p/assets/5486389/954b00cd-47e0-4e07-941d-c04df3f0796d)

In fact, there are two because we add our own one here:
https://github.com/libp2p/rust-libp2p/blob/38ea7ad4534565876225454a71edf91ce67adb16/.github/ISSUE_TEMPLATE/config.yml#L3-L5

I think we can remove this text here and the duplicated link in favor of the link that GitHub adds automatically.

Do you agree @mxinden?